.github/stale.yml

```diff
@@ -0,0 +1,60 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 2
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+  - security
+  - '[Status] Maybe Later'
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
```

Comment:
Does `false` mean this will also close stale issues? We have some issues that have been around for quite long and it does not look like we are ready to close them.

Reply:
let's use a new label `backlog` for such cases. WDYT?

Reply:
In that case should we exclude issues with `backlog` tag too? Or do you want to close those?

Reply:
yeah, I've done exactly that, added `backlog` and `dependencies` labels to that list.